scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontendState.java

```diff
@@ -344,6 +344,10 @@ synchronized SchedulerState getState() throws NotConnectedException, PermissionE
         return getState(false);
     }
 
+    synchronized SchedulerState getStateInternally() {
+        return sState;
+    }
+
     synchronized SchedulerState getState(boolean myJobsOnly) throws NotConnectedException, PermissionException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)